src/test/bundle_test.ts

```diff
@@ -391,4 +394,108 @@ suite('BuildBundler', () => {
     // We should not have the inlined file in the output.
     assert.isNotOk(getFile('framework.html'));
   });
+
+  suite('options', () => {
+
+    const projectOptions = {
+      root: 'test-fixtures/test-project',
+      entrypoint: 'index.html',
+      fragments: ['shell.html'],
+      sources: ['index.html', 'shell.html', 'source-dir/my-app.html']
+    };
+
+    test('excludes: a file', async() => {
```

Comment:
I'll update the test names to include description of expected behavior. @FredKSchott @justinfagnani:

```
options
      ✓ excludes: html file urls listed are not inlined (51ms)
      - excludes: html files in folders listed are not inlined
      ✓ excludes: nothing is excluded when no excludes are given (40ms)
      ✓ inlineCss: false, does not inline external stylesheets
      ✓ inlineCss: true, inlines external stylesheets (41ms)
      ✓ inlineScripts: false, does not inline external scripts
      ✓ inlineScripts: true, inlines external scripts
      ✓ stripComments: false, does not strip html comments
      ✓ stripComments: true, strips html comments
      ✓ strategy: fn(), applies bundle strategy function
      ✓ urlMapper: fn(), applies bundle url mapper function
```